src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/isconnected.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 'isConnected()'
+excerpt: 'xk6-browser: Browser.isConnected method'
+---
+
+<Blockquote mod="warning">
+
+There is a known issue with this feature. See [issue #453](https://github.com/grafana/xk6-browser/issues/453) for details.
+
+</Blockquote>
+
+Indicates whether the [CDP](https://chromedevtools.github.io/devtools-protocol/) connection to the browser process is active or not.
+
+### Returns
+
+| Type    | Description                                                                                    |
+| ------- | ---------------------------------------------------------------------------------------------- |
+| boolean | Returns `true` if Browser is connected to the browser application. Otherwise, returns `false`. |
```

Comment:
```suggestion
| boolean | Returns `true` if [Browser](/javascript-api/xk6-browser/browser/) is connected to the browser application. Otherwise, returns `false`. |
```